frame/support/procedural/src/construct_runtime/expand/origin.rs

```diff
@@ -68,14 +68,39 @@ pub fn expand_outer_origin(
 	}
 
 	let system_path = &system_pallet.path;
+
 	let system_index = system_pallet.index;
 
+	let system_path_name = system_path.module_name();
+
+	let doc_string = get_intra_doc_string(
+		"Origin is always created with the base filter configured in",
```

Comment:
One last question, so this leading whitespace was never needed? I would have thought that you may need to add it in your function below, but there is already one.

I just want to have this clarified and then I'm going to merge this.

Reply:
Yes... The leading whitespace was never needed. You can go ahead and merge